util/rlp/fuzz/fuzz_targets/append_raw.rs

```diff
@@ -0,0 +1,19 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+#[cfg(feature = "nightly")]
```

Comment:
figured out enabling features in build scripts:
`println!("cargo:rustc-cfg=feature=\"nightly\"")`